tests/script/telemetry/CrashCounter.py

```diff
@@ -0,0 +1,121 @@
+from datetime import date
+from datetime import timedelta
+import os
+import re
+import requests
+
+# define version
+version = "3.3.*"
+ip = "103.229.218.146"
```

Comment:
The IP and notification URL should not be visible in the script, instead they can be loaded from env variables.

Reply:
Done